config/features.yml

```diff
@@ -798,16 +798,19 @@ features:
     actor_type: user
     description: When enabled, profile service history will include military academy attendance.
     enable_in_development: true
-  profile_hide_direct_deposit_comp_and_pen:
```

Comment:
Once merged and deployed, you'll need to delete this old name (`Flipper.remove(:feature_name)`) in the rails console for dev, staging, sandbox and prod. See step three here: https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Removingold/unusedfeaturetoggles

Reply:
Yep thanks for the callout! @tpharrison is gonna handle that once this is merged.